tasks/collect-slack-notification-params/collect-slack-notification-params.yaml

```diff
@@ -0,0 +1,180 @@
+---
+apiVersion: tekton.dev/v1
+kind: Task
+metadata:
+  name: collect-slack-notification-params
+  labels:
+    app.kubernetes.io/version: "0.0.1"
+  annotations:
+    tekton.dev/pipelines.minVersion: "0.12.1"
+    tekton.dev/tags: release
+spec:
+  description: >-
+    Tekton task that collects slack notification configuration options from Release CRs and the data file
+  params:
+    - name: release
+      type: string
+      description:
+        The namespaced name (namespace/name) of the Release custom resource initiating this pipeline execution
+    - name: releaseplanadmission
+      type: string
+      description: The namespaced name (namespace/name) of the releasePlanAdmission
+    - name: releasepipelinename
+      description: name of the release pipeline
+      type: string
+    - name: snapshotPath
+      description: Path to the JSON string of the mapped Snapshot spec in the data workspace
+      type: string
+      default: "mapped_snapshot.json"
+    - name: dataPath
+      type: string
+      description: Path to the JSON string of the merged data containing the configuration options to use
+      default: data.json
+    - name: hacUrl
+      type: string
+      description: Root URL of HAC UI (hybrid access console)
+      default: "https://console.redhat.com/preview/application-pipeline/workspaces"
+  workspaces:
+    - name: data
+      description: The workspace where the snapshot spec json file resides
+  results:
+    - name: message
```

Comment:
I'm confused. The task name says it is collecting notification params, but only a message is output? Where are the other params

Reply:
right now, `message` is going to be the only param to be passed to the `send` task

Reply:
let me know if you want me to rename the entire task accordingly...

Reply:
no, that's fine